pkg/pod/script.go

```diff
@@ -62,13 +62,15 @@ func convertScripts(shellImage string, steps []v1beta1.Step, sidecars []v1beta1.
 	}
 
 	convertedStepContainers := convertListOfSteps(steps, &placeScriptsInit, &placeScripts, "script")
+
 	// convertListOfSteps operates on overlapping fields across Step and Sidecar, hence a conversion
 	// from Sidecar into Step
 	sideCarSteps := []v1beta1.Step{}
 	for _, step := range sidecars {
 		sidecarStep := v1beta1.Step{
 			step.Container,
 			step.Script,
```

Comment:
This should be:

```
sidecarStep := v1beta1.Step{
  Container: step.Container,
  Script:    step.Script,
}
```

(This makes it clearer what fields these values are being passed to, rather than depending on the field ordering in the struct, which could change subtly in the future.)